packages/storage-documents/src/page-scan-run-result-map.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export declare type DocumentType = 'pageScanRunResult' | 'pageScanRunSarifReport';
+
+/**
+ * The mapping document between scan run ID and all scan run corresponding documents.
+ */
+export interface PageScanRunResultMap {
+    scanId: string;
+    url: string;
+    links: Link[];
```

Comment:
'link' sounds a bit too general?

Reply:
This is what it is - link to other files location